pkgs/development/interpreters/dart/default.nix

```diff
@@ -25,6 +25,14 @@ let
       url = "${stable}/${version}/sdk/dartsdk-linux-ia32-release.zip";
       sha256 = "d67b8f8f9186e7d460320e6bce25ab343c014b6af4b2f61369ee83755d4da528";
     };
+    "2.0.0-x86_64-linux" = fetchurl {
+      url = "{$stable}/${version}/sdk/dartsdk-linux-x64-release.zip";
```

Comment:
You have a small syntax error there ;)

The `$` of the `stable` part isn't where it should be.